### PR TITLE
lightning: update configure about backslash

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)
Add more details about backslash configuration. 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4068
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ recognized and transformed:
     | `\Z`     | Windows EOF (U+001A)     |
 
     In all other cases (e.g. `\"`) the backslash is simply stripped, leaving the next character (`"`)
-in the field.
+in the field. Meanwhile, the reserved characters are only ordinary characters, and all special characters (e.g. delimiter) will be invalid.
```
Comment (TomShawn) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:125]:
```suggestion
in the field. In these situations, the reserved characters are only ordinary characters, and their special usages (for example, delimiters) will be unavailable.
```

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ recognized and transformed:
     | `\Z`     | Windows EOF (U+001A)     |
 
     In all other cases (e.g. `\"`) the backslash is simply stripped, leaving the next character (`"`)
```
Comment (TomShawn) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:124]: Please move L125 at the end of the L124.
```suggestion
    In all other cases (for example, `\"`) the backslash is simply stripped, leaving the next character (`"`)
```

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 3]
```diff
@@ -121,8 +121,7 @@ recognized and transformed:
     | `\t`     | Tab (U+0009)             |
     | `\Z`     | Windows EOF (U+001A)     |
 
-    In all other cases (e.g. `\"`) the backslash is simply stripped, leaving the next character (`"`)
-in the field.
+    In all other cases (for example, `\"`) the backslash is simply stripped, leaving the next character (`"`)in the field. In these situations, the reserved characters are only ordinary characters, and their special usages (for example, delimiters) will be unavailable.
```
Comment (kennytm) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:124]:
```suggestion
    In all other cases (for example, `\"`) the backslash is simply stripped, leaving the next character (`"`) in the field. In these situations, the reserved characters are only ordinary characters, and their special usages (for example, delimiters) will be unavailable.
```

**Comment by kennytm** [comment]
LGTM about the technical details, but the wording sounds strange.

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 4]
```diff
@@ -121,8 +121,7 @@ recognized and transformed:
     | `\t`     | Tab (U+0009)             |
     | `\Z`     | Windows EOF (U+001A)     |
 
-    In all other cases (e.g. `\"`) the backslash is simply stripped, leaving the next character (`"`)
-in the field.
+    In all other cases (for example, `\"`) the backslash is simply stripped, leaving the next character (`"`) in the field. In these situations, the reserved characters are only ordinary characters, and their special usages (for example, delimiters) will be unavailable.
```
[the remark this answers is not in the document]
Comment (kennytm) [reply tidb-lightning/migrate-from-csv-using-tidb-lightning.md:124]: Lgtm

**Pull request merged**